filebeat/docs/filebeat-log-rotation.asciidoc

```diff
@@ -0,0 +1,89 @@
+[[input-file-log-rotation]]
+== Configure logrotation for the input files of {beatname_uc}
```

Comment:
I have quite a few editorial changes I'd like to make to this section. It will be easier for me to make the changes directly to the source files. How do you want me to proceed? Should I push commits to your branch, or wait until you've merged into master?

If you like the idea of moving this to the troubleshooting section and want me to push commits to your branch, I can also take care of moving the file. Let me know.